app/controllers/posts_controller.rb

```diff
@@ -485,6 +487,9 @@ def notice
       post.delete_post_notices
     end
 
+    details = { new_raw_value: params[:notice], old_value: previous_notice }
+    StaffActionLogger.new(current_user).log_post_staff_note(post, details)
```

Comment:
I think we should update the request specs for this controller action to ensure that the staff action is logged too.

Reply:
Done